[2025-09-21T12:18:16.686+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T12:14:00+00:00/task_id=validate_model_performance permission to 509
[2025-09-21T12:18:16.687+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T12:14:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T12:14:00+00:00/task_id=validate_model_performance'
[2025-09-21T12:18:16.735+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T12:14:00+00:00/task_id=validate_model_performance permission to 509
[2025-09-21T12:18:16.738+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T12:14:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T12:14:00+00:00/task_id=validate_model_performance'
[2025-09-21T12:18:16.804+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-09-21T12:14:00+00:00 [queued]>
[2025-09-21T12:18:16.816+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-09-21T12:14:00+00:00 [queued]>
[2025-09-21T12:18:16.817+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2025-09-21T12:18:16.835+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): validate_model_performance> on 2025-09-21 12:14:00+00:00
[2025-09-21T12:18:16.847+0000] {standard_task_runner.py:57} INFO - Started process 222 to run task
[2025-09-21T12:18:16.859+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'train_predict_to_db', 'validate_model_performance', 'scheduled__2025-09-21T12:14:00+00:00', '--job-id', '201', '--raw', '--subdir', 'DAGS_FOLDER/train_predict_to_db_dag.py', '--cfg-path', '/tmp/tmpzslimomq']
[2025-09-21T12:18:16.863+0000] {standard_task_runner.py:85} INFO - Job 201: Subtask validate_model_performance
[2025-09-21T12:18:16.935+0000] {logging_mixin.py:151} INFO - Changing /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T12:14:00+00:00/task_id=validate_model_performance permission to 509
[2025-09-21T12:18:16.936+0000] {logging_mixin.py:151} INFO - Failed to change /opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T12:14:00+00:00/task_id=validate_model_performance permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=train_predict_to_db/run_id=scheduled__2025-09-21T12:14:00+00:00/task_id=validate_model_performance'
[2025-09-21T12:18:16.938+0000] {task_command.py:416} INFO - Running <TaskInstance: train_predict_to_db.validate_model_performance scheduled__2025-09-21T12:14:00+00:00 [running]> on host 1bfed80107dd
[2025-09-21T12:18:17.051+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='train_predict_to_db' AIRFLOW_CTX_TASK_ID='validate_model_performance' AIRFLOW_CTX_EXECUTION_DATE='2025-09-21T12:14:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-21T12:14:00+00:00'
[2025-09-21T12:18:44.745+0000] {python.py:194} INFO - Done. Returned value was: {'mse': 840546.5943688372, 'rmse': 916.8132821730045, 'mae': 801.1260952419705, 'r2': 0.8767890759049677}
[2025-09-21T12:18:44.933+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=train_predict_to_db, task_id=validate_model_performance, execution_date=20250921T121400, start_date=20250921T121816, end_date=20250921T121844
[2025-09-21T12:18:45.098+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-09-21T12:18:45.157+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
